functions/firestore/index.js

```diff
@@ -0,0 +1,17 @@
+const { firestore, config } = require('firebase-functions');
+
+const { sendEmail } = require('../modules/ses');
+
+const { EXTERNAL_URL } = process.env;
+
+module.exports = firestore
+  .document(`${config().db.firestore_nft_mint_subscriptions_root}/{id}`)
+  .onCreate(snapshot => {
+    const subscriptionId = snapshot.id;
+    const { subscriberEmail, subscribedWallet } = snapshot.data();
+    return sendEmail({
+      email: subscriberEmail,
+      subject: 'Writing NFT - NFT Subscription',
+      body: `Hi you have subscribed ${subscribedWallet}, you can cancel the subscription by ${EXTERNAL_URL}/${subscribedWallet}/unsubscribe/${subscriptionId}`,
```

Comment:
Can we somehow make a global token for a email to unsubscribe all wallet?

Reply:
Maybe we implement this in session feature?